config.go

```diff
@@ -100,11 +114,16 @@ type CloudSQLConfig struct {
 
 // File is a collection of jobs
 type File struct {
+	Misc           Misc              `yaml:"misc,omitempty"`
```

Comment:
Two questions:
- What do you think about calling it Configuration or something to make it more explicit that it's a new place to put advanced configuration options.
- Would it make sense to have this part job specific and not globally available? I could imagine that there's maybe jobs that take longer where you want different buckets than just for a quick `select`

Reply:
> What do you think about calling it Configuration or something to make it more explicit that it's a new place to put advanced configuration options.

LGTM

> Would it make sense to have this part job specific and not globally available? I could imagine that there's maybe jobs that take longer where you want different buckets than just for a quick select

If my understanding is correct the bucket definition is done at the histogram definition. If we want to have the bucket definition at the job level we need to define multiple histogram which can severely impact the cardinality. I would recommend waiting for the native histogram which are design to address such issue.